mta-otp-proxy/src/main/resources/beans/common.xml

```diff
@@ -10,6 +10,7 @@
     <!-- can split off to separate beans if different alert feeds require different white lists -->
     <bean name="siriDeserializerWithModifications" class="com.camsys.shims.gtfsrt.alerts.siri.deserializer.SiriDeserializerWithModifications">
         <property name="htmlCleanupUtil" ref="htmlCleanupUtil" />
+        <property name="filterLmm" value="True" />
```

Comment:
I've never use this capitalization format in Java/Spring.  Does it work?

Reply:
I was using it as a string but I switched to lowercase and use it as boolean. That will be less confusing.